ViAn/Test/test_video_player.cpp

```diff
@@ -246,3 +226,17 @@ void test_video_player::test_set_stop_video() {
     QVERIFY(!mvideo->video_paused);
     QVERIFY(mvideo->video_stopped);
 }
+
+/**
+ * @brief test_video_player::test_on_set_playback_fram
+ */
+void test_video_player::test_on_set_playback_fram() {

```

Comment:
frame is misspelled in the method name "fram". Also it could be good to test a frame number that is negative or larger than the total amounts of frames in the video.